src/Contributing/index.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-As Vinegar is open source, contributions with documentation, translations, and code are greatly appreciated!
+Since Vinegar is open source, contributions with documentation, translations and code are greatly appreciated!
```

Comment:
```suggestion
Since Vinegar is open source, contributions with documentation, translations, and code are greatly appreciated!
```
I would keep the Oxford comma here.